Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.6.6
+
+RUN apt-get update && \
+  apt-get install --reinstall -y locales && \
+  sed -i 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+  locale-gen en_US.UTF-8
```

Comment:
Is all of this necessary?

Reply:
Apparently. If i remove it or remove the `sed` part it doesn't seem to set the language to UTF-8 properly and SASS fails:
```
ruby_1  |            Assets:  Invalid US-ASCII character "\xEF"
ruby_1  |          Asset File core/mixins/_utility-builder.scss:382
ruby_1  |   Liquid Exception: Sass Error in /_layouts/base.html
```
I actually stole this from our build container.

I haven't researched enough to know WHY this is this case...